code/game/mecha/mecha_parts.dm

```diff
@@ -13,6 +13,11 @@
 	if(!user.transferItemToLoc(src, M))
 		to_chat(user, span_warning("\The [src] is stuck to your hand, you cannot put it in \the [M]!"))
 		return FALSE
+	
+	// If there is an occupant and it is not a posibrain
+	if(M.occupant && !istype(M.occupant.loc, /obj/item/mmi/posibrain)) 
```

Comment:
```suggestion
	if(!istype(M?.occupant?.loc, /obj/item/mmi/posibrain)) 
```

Reply:
if(!istype(M?.occupant?.loc, /obj/item/mmi/posibrain))
if there's no occuptant, it would return false, which would return true because of the !
which would deny putting in parts if there's no occupant
which is the opposite of what this PR aims